quantmsutils/diann/diann2mztab.py

```diff
@@ -465,7 +460,17 @@ def main_report_df(self, qvalue_threshold: float) -> pd.DataFrame:
             "Global.PG.Q.Value",
             "MS2.Scan",
         ]
-        report = pd.read_csv(self.report, sep="\t", header=0, usecols=remain_cols)
+        if ".tsv" in self.report.suffix:
+            report = pd.read_csv(self.report, sep="\t", header=0, usecols=remain_cols)
+        else:
+            report = pd.read_parquet(self.report)
+        # filter decoy
+        if "Decoy" in report.columns:
```

Comment:
@daichengxin in quantms OpenMS, we do have the possibility to export to msstats and mzTab the decoys features. This is what the following variable does in quantms `quantify_decoys`. Let's do the same now that they export them.